README.md

```diff
@@ -9,14 +9,46 @@ A simple watch files utility
 You can use it to watch files and run any command.  
 It is not necessary to be a Golang project.  
 
+## Installation
+
 ```bash
 $ go get -u github.com/rafaelsq/wtc
+```
+
+## Usage
+
+```
+Watch files
+If you have a .wtc.yaml or wtc.yaml file it will be used instead of the arguments
+
+Usage:
+  wtc [command] [arguments] [flags]
+
+Flags:
+  -d, --debounce int     (default 300)
+  -h, --help            help for wtc
+  -i, --ignore string   regex to be used (default "\\.git")
+      --no_trace        do not show the command being executed
+```
+
+### Quickstart
+
+If you run `wtc [flags] "<build-cmd>" "<run-cmd>"`, it will replace default `command`s above. 
+
+#### Example
+
+```
 $ cd my_go_project
 $ wtc "go build main.go" "./my_go_project"
 ```
 
+### Using a configuration file
+
 You can create an Yaml file with your rules.
-Default;
+If you create your own `.wtc.yaml` or `wtc.yaml`, no default rules will exists.
+
+Default:
```

Comment:
You can remove this `default`.
Maybe replace with some example like;
`wtc ".go$" "go test -cover {PKG}"`